Eevee.Sleep.Bot/Modules/SlashCommands/ChangeRoleSlashModule.cs

```diff
@@ -0,0 +1,64 @@
+using Discord;
+using Discord.Interactions;
+using JetBrains.Annotations;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Utils;
+
+namespace Eevee.Sleep.Bot.Modules.SlashCommands;
+
+[Group("role", "Commands for role things.")]
```

Comment:
"Commands for managing the roles on a user."

Reply:
Included here
[042bf5a](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/042bf5ae19def3a76b4fa219af2021bdffdd459e)